Test of subdirectory creation within c_examples

ChrisHird opened the pull request:
I am testing a new branch plus adding sub directory creation with IProjects in RDi.

Commit:
Test of subdirectory creation within c_examples

worksofliam commented:
Hey @ChrisHird 

If you check out `HELLOWORLD.c` in the commit, it has the numeric data at the beginning of each line from the QRPGLESRC file - SRCSEQ and SRCDAT.

I'm afraid we won't be able to accept this PR until they are removed from the source.

ChrisHird commented:
Ahhh

Forgot to remove before publishing. Line numbers from source file.
Just testing new branch pull request. Not a problem for the IBM i, just need it for readability on github?

Sent from my iPhone

> On Dec 10, 2016, at 7:19 AM, Liam Allan <notifications@github.com> wrote:
> 
> Hey @ChrisHird
> 
> If you check out HELLOWORLD.c in the commit, it has the numeric data at the beginning of each line from the QRPGLESRC file - SRCSEQ and SRCDAT.
> 
> I'm afraid we won't be able to accept this PR until they are removed from the source.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or mute the thread.
>

worksofliam commented:
Chris

Yes, just for readability and portability in general. If you simply commit a change to your fork (ChrisHird/OSSILE) they will be picked up in this PR.

Commit:
Adding back the test file with no sequence numbers

Commit:
removed test file

Commit:
Adding ftpclient to main

Commit:
Delete test

Commit:
Delete test example

ChrisHird committed:
Create readme.txt

ChrisHird committed:
Update readme.txt

Commit:
Added README.md

Commit:
Added build.txt

Commit:
Initial upload of the ZLIB project

Commit:
initial push of MD5 CRC build tool

Commit:
Push of sample 'C' code Check OS Level

ThePrez commented:
Hi, Chris. Thanks for your contributions! However, upon initial review, I've noted the following:
1. The ZLIB package has its own license. This will need to be a secondary license for the project. I'll look into it, but this should be ok
2. The ftp client is licensed under the GPL. GPL-licensed code is not allowed in the OSSILE project. Did you create this code? If so, do you agree to re-licensing the code under the terms of the MIT license?
3. It is lacking a setup script as documented in CONTRIBUTING.md. Can you please work on this? 

Thank you!

ThePrez reviewed (changes requested):
I love this utility! Does something very similar to what an IBM internal tool does (I always wished customers had it)

Could we move this over to the main/ portion of the product and wrap the compiles around it? So people could do "CALL OSSILE/DSPOSVER" or somesuch. 

Thanks!

ChrisHird commented:
Jesse

 

I am sure the secondary license will be OK, the one it ships with is very loose. Just need to ensure people don’t remove the license from the files.

I own ftpclient its all my own code, I just released it un GPL originally so it would be out there. We can change the license to meet the OSSILE needs.

There is a build.txt file which is how we shipped originally to install with Relic??? I was under the assumption we could use the same install procedure, but if not then we can look at adjusting the setup script to allow it to be installed. I will ask Liam to give me a hand, he needs something to do :)

 

Chris..

From: ThePrez [mailto:notifications@github.com] 
Sent: December 12, 2016 5:26 PM
To: OSSILE/OSSILE <OSSILE@noreply.github.com>
Cc: Chris Hird <chrish@shieldadvanced.com>; Mention <mention@noreply.github.com>
Subject: Re: [OSSILE/OSSILE] Test of subdirectory creation within c_examples (#30)

 

Hi, Chris. Thanks for your contributions! However, upon initial review, I've noted the following:

1.	The ZLIB package has its own license. This will need to be a secondary license for the project. I'll look into it, but this should be ok
2.	The ftp client is licensed under the GPL. GPL-licensed code is not allowed in the OSSILE project. Did you create this code? If so, do you agree to re-licensing the code under the terms of the MIT license?
3.	It is lacking a setup script as documented in CONTRIBUTING.md. Can you please work on this?

Thank you!

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub <https://github.com/OSSILE/OSSILE/pull/30#issuecomment-266572935> , or mute the thread <https://github.com/notifications/unsubscribe-auth/ARZ28TUFmScIWbQXyt6XcDXQR03VYuCWks5rHcoCgaJpZM4LJfxu> .

ChrisHird commented:
Jesse,

 

Lets set up a project and get working on it. I will do what I can, just need a bit of guidance on how we want to build things. Projects should allow many to get involved instead of me just pushing up my code and then changing everything before shoving it all back up again? At least that’s how I read it should work??? I have got so many sample utilities I built it is going to be difficult to get them all tidied up (I do quick and dirty code to get things up and running so I need to improve the code). I would also like to start a real ILE project that creates a set of modules and service programs that people can download install and then use the functionality from the service programs. Its going to show people the real power of ILE as we can have people add their own language based module and ship them all as a complete service program.. Showing how to use binding directories and exporting via a source member, making sure they understand signatures etc etc etc..

 

Chris…

 

From: ThePrez [mailto:notifications@github.com] 
Sent: December 12, 2016 5:33 PM
To: OSSILE/OSSILE <OSSILE@noreply.github.com>
Cc: Chris Hird <chrish@shieldadvanced.com>; Mention <mention@noreply.github.com>
Subject: Re: [OSSILE/OSSILE] Test of subdirectory creation within c_examples (#30)

 

@ThePrez requested changes on this pull request.

I love this utility! Does something very similar to what an IBM internal tool does (I always wished customers had it)

Could we move this over to the main/ portion of the product and wrap the compiles around it? So people could do "CALL OSSILE/DSPOSVER" or somesuch.

Thanks!

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub <https://github.com/OSSILE/OSSILE/pull/30#pullrequestreview-12566277> , or mute the thread <https://github.com/notifications/unsubscribe-auth/ARZ28apsJkOi0m4SSf3JJWV7D4N-f4-9ks5rHcuCgaJpZM4LJfxu> .

Commit:
Added setup.txt

Commit:
Added setup.txt to md5_checksum

Commit:
Added new Receiver clean up program.

ThePrez reviewed (changes requested):
Some comments on the setup script:
1) should build everything into the OSSILE library
2) file should be named "setup" instead of "setup.txt"

Please see our CONTRIBUTING.md for (minimal) guidance: https://github.com/OSSILE/OSSILE/blob/master/CONTRIBUTING.md

ThePrez reviewed (dismissed):
Same notes apply as to the other setup.txt in the pull request. Thank you!

Commit:
New code plus rename of setup file

Commit:
Disk Util added that shows how to extract system and dasd information

Commit:
Added sample to display constraints

Commit:
Added program to display triggers attached to file

Commit:
Added sample which lists the database relationships for file

Commit:
Added new source code which displays the objects journalled to a
journal.

Commit:
Added a sample to show how to compare signatures between ILE *PGM and
*SRVPGM

Commit:
Added a sample program to carry out backups to IMGCLG 

Using the IBM supplied backup process and IBM job scheduler it will
carry out the required save before copying the objects to a remote file
server.

Commit:
Cleanup

Commit:
claenup2
